index.html

```diff
@@ -11,33 +11,3186 @@
         shortName: "did-rubric",
         edDraftURI: "https://w3c.github.io/did-rubric/",
         wgPublicList: "public-did-wg",
+        wg: "Decentralized Identifier Working Group",
+        wgURI: "https://www.w3.org/2019/did-wg/",
         wgPatentURI:  "https://www.w3.org/2004/01/pp-impl/117488/status",
         github:                  {
             repoURL: "https://github.com/w3c/did-rubric/",
             branch: "master"
         },
-        noRecTrack: "true"
+        noRecTrack: "true",
+        editors: [{
+          name: "Joe Andrieu"
+        }],
+        authors: [{
+          name: "Joe Andrieu"
```

Comment:
Ditto. Please use information from above.

Reply:
I'll propose that I be included as an author when I submit a meaningful content PR.

Reply:
That sounds right.